source/app.d

```diff
@@ -62,11 +62,21 @@ void defaultInit(UserManController userMan, DubRegistry registry)
 		registry.getPackageDump().empty && userMan.getUserCount() == 0)
 	{
 		logInfo("'GENERATE_DEFAULT_DATA' is set and an empty database has been detected. Inserting dummy data.");
-		auto userId = userMan.registerUser("dummy@dummy.org", "dummyUser",
+		auto userId = userMan.registerUser("dummy@dummy.org", "dummyuser",
```

Comment:
The registry automatically lowercases usernames, but not when searching for them on login. This should just reduce confusion.